packages/block-library/src/site-logo/index.php

```diff
@@ -80,6 +80,25 @@ function register_block_core_site_logo_setting() {
 
 add_action( 'rest_api_init', 'register_block_core_site_logo_setting', 10 );
 
+/**
+ * Register a core site setting for a site icon
+ */
+function register_block_core_site_icon_setting() {
+	register_setting(
+		'general',
+		'site_icon',
+		array(
+			'show_in_rest' => array(
+				'name' => 'site_icon',
```

Comment:
Could this be `'show_in_rest' => true,` since we're not renaming the `$option_name` param?

Reply:
Also I think we need to add the translation for `__( 'Site icon' )` in the `TRANSLATED_SITE_PROTPERTIES` [sic] list here: https://github.com/WordPress/gutenberg/blob/9df736c678b2e8ef00c355e6cc9fb8092fe25484/packages/editor/src/components/entities-saved-states/index.js#L25

Otherwise we see the `site_icon` property name.
<img width="277" alt="Screen Shot 2021-11-01 at 9 40 20 am" src="https://user-images.githubusercontent.com/6458278/139603862-b30504b0-cefa-4446-84e7-772c0353dd38.png">